runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -384,10 +384,10 @@ NNFW_STATUS nnfw_session::prepare()
   try
   {
     _tracing_ctx = std::make_unique<onert::util::TracingCtx>(_subgraphs.get());
-    _compiler =
+    auto compiler =
       std::make_unique<onert::compiler::Compiler>(_subgraphs, _tracing_ctx.get(), *_coptions);
```

Comment:
At first, I was going to remove `_tracing_ctx` from Compiler constructor argument. But it turned out that `_tracing_ctx` is in/out argument. As input, it doesn't need to be passed because it only requires `_subgraphs` to generate initial map, and `_subgraphs` is already a member of `Compiler`. But it is also there to get output (modified map during lowering: LoweredGraph to subgraph ID) after `compiler()`.

Reply:
I have one suggestion. Can we generate `_tracing_ctx` on Compiler.compile()`?